spec/board_spec.rb

```diff
@@ -1,20 +1,72 @@
 require_relative "../lib/board"
 
 describe "Board" do
+  let(:game_checker) { GameChecker.new }
+  subject(:board) { Board.new(game_checker) }
+
```

Comment:
This is not truly a unit test, as you pass in the real GameChecker.  Can you make this a unit test instead of an integration test?

Reply:
So I was having some trouble making a TestChecker that responded correctly to both cases of Board.won?. Should I just delete the tests for Board.won? and Board.tie?, since they really just return the results of the GameChecker functions?

Reply:
GameChecker returns a boolean.  Even though it gets two arguments, you don't have to do anything with the arguments.  Just return what you want to return.  You could create a couple of TestCheckers returning different values, one could be TestCheckerThatAlwaysReturnsTrue, and the other could be TestCheckerThatAlwaysReturnsFalse.  For example,
```
  def winner?(board, player)
    true
  end
```

Reply:
Interesting.  You should still test the won and tie methods.  You could also check for the mark to either return true or false in the mocks.